ros2/README.md

```diff
@@ -0,0 +1,185 @@
+# ROS 2 Bindings
+
+This document defines how to describe ROS 2-specific information in AsyncAPI.
+
+It applies to all versions of ROS 2 (foxy, galactic, humble, iron, jazzy).
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+<a name="server"></a>
+
+## Server Binding Object
+
+This object contains information about the server representation in ROS 2. 
+ROS 2 can use either DDS or Zenoh as its middleware. 
+DDS is decentralized with no central server, so the field `host` can set to `localhost`.
+When using Zenoh, the `host` field specifies the Zenoh Router IP address.
+
+###### Fixed Fields
+
+Field Name | Type | ROS 2 Versions | Description
+---|:---:|:---:|---|
+`rmwImplementation` | string  | all | Specifies the ROS 2 middleware implementation to be used. Valid values include `rmw_fastrtps_cpp` (Fast DDS), `rmw_cyclonedds_cpp` (Cyclone DDS), `rmw_connext_cpp` (RTI Connext), and `rmw_zenoh_cpp` (Zenoh). This determines the underlying middleware implementation that handles communication.
+`domainId` | integer  | all | All ROS 2 nodes use domain ID 0 by default. To prevent interference between different groups of computers running ROS 2 on the same network, a group can be set with a unique domain ID. Must be a non-negative integer less than 232. 
```

Comment:
```suggestion
`domainId` | integer  | all | All ROS 2 nodes use domain ID 0 by default. To prevent interference between different groups of computers running ROS 2 on the same network, a group can be set with a unique domain ID. [Must be a non-negative integer less than 232](https://docs.ros.org/en/rolling/Concepts/Intermediate/About-Domain-ID.html).
```

> domain IDs 0-101 and 215-232 can be safely used without colliding with ephemeral ports. 

This is why I would add the link

Reply:
solved